.storybook/stories/paragraphs/Gallery.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {ComponentProps} from "react";
+import GalleryParagraph from '@components/paragraphs/stanford-gallery/gallery-paragraph';
+import { getStoryBookGalleryImage } from '../storybook-entities';
+
+type ComponentStoryProps = ComponentProps<typeof GalleryParagraph> & {
+
+}
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta: Meta<ComponentStoryProps> = {
+  title: 'Design/Paragraphs/Gallery',
+  component: GalleryParagraph,
+  tags: ['autodocs'],
+  argTypes: {
+    numberofimages: {
+      control: {
+        type: 'number',
+        min: 1,
+        max: 20,
+      }
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<ComponentStoryProps>;
+
+// More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+export const Gallery: Story = {
+  render: ({numberofimages, ...args}) => {
```

Comment:
@pookmish I added the number of images here hoping to use the selected number further down on the `suGalleryImages`